lang/fr/lang.php

```diff
@@ -2,15 +2,16 @@
 
 /**
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
- * 
+ *
  * @author Schplurtz le Déboulonné <schplurtz@laposte.net>
  * @author Nicolas Friedli <nicolas@theologique.ch>
  * @author tuxun <tuxuntrash@gmail.com>
- * @author Schplurtz le Déboulonné <Schplurtz@laposte.net>
+ * @author ubibene <services.m@benard.info>
  */
 $lang['export_pdf_button']     = 'Exporter en PDF';
 $lang['needtitle']             = 'Veuillez choisir un titre.';
 $lang['needns']                = 'Veuillez choisir une catégorie existante.';
 $lang['empty']                 = 'Vous n\'avez pas encore sélectionné de pages.';
 $lang['tocheader']             = 'Table des matières';
 $lang['export_ns']             = 'Exporter la catégorie  « %s » dans le fichier %s.pdf.';
+$lang['forbidden']             = 'Vous ne pouvez pas accéder à ces pages: %s.<br/><br/>Utilisez l\'option  \'Passer les Pages Interdites\' pour créer votre livre avec les pages disponibles.';
```

Comment:
There is a missing space here. It should be "à ces pages :" (space between "pages" and ":").